libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.ts

```diff
@@ -117,11 +119,26 @@ export class DhMessageArchiveLogSearchComponent implements OnDestroy {
         ? q.get('invocationId')
         : null;
     });
+  }
 
-    this.searchCriteria.dateTimeFrom =
-      this.initDateFrom().toISOString().split('.')[0] + 'Z';
-    this.searchCriteria.dateTimeTo =
-      this.initDateTo().toISOString().split('.')[0] + 'Z';
+  private buildRsmOptions() {
+    const entries = Object.entries(DocumentTypes);
+    entries.forEach((entry) => {
+      this.rsmFormFieldOptions.push({
+        value: entry[0],
+        displayValue: entry[1] + ' - ' + entry[0],
+      });
+    });
+  }
```

Comment:
suggestion: instead of modifying the `rsmFormFieldOptions` property by pushing array items to it, consider building the array in the method and returning it. This way you keep the function more "pure". So it could be something like:

```suggestion
  private buildRsmOptions() {
    return Object.entries(DocumentTypes).map((entry) => {
      return {
        value: entry[0],
        displayValue: entry[1] + ' - ' + entry[0],
      };
    });
  }
```